app/models/service_template.rb

```diff
@@ -375,7 +375,9 @@ def self.create_from_options(options)
   private_class_method :create_from_options
 
   def provision_request(user, options = nil, request_options = nil)
-    provision_workflow(user, options, request_options).submit_request
+    result = provision_workflow(user, options, request_options).submit_request
+    raise result[:errors].join(",") if result[:errors].any?
```

Comment:
Minor, but please add a space after the comma to separate the data a little bit.  `.join(", ")`